package.json

```diff
@@ -38,6 +38,7 @@
     "raw-loader": "^4.0.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "sharp": "^0.33.4",
```

Comment:
Unsure if related to the build failure, but is this diff expected?

Reply:
Ah, maybe! I had to override that package because it had issues with my machine. 

I can strike it for the PR, if it's not impacting others.

Reply:
@beriberikix sounds good!